project_kanban/models/project.py

```diff
@@ -16,4 +16,6 @@ def _compute_get_recent_date(self):
             project.recent_date = date_lst and max(date_lst) or False
 
     recent_date = fields.Datetime(compute="_compute_get_recent_date",
-                                  string="Recent date")
+                                  string="Recent date",
+                                  help="This will allow you to add \
+                                  Recent Date")
```

Comment:
Tooltip should be:
'This is an auto-set date based on Last recent change on Project'